### DISCUSSION TITLE
Have vite-ssr not render certain components/paths

### DISCUSSION THREAD

**Discussion by LagnerKol** [frandiox/vite-ssr#94] [Q&A] [answered]
Hi, i checked the docs and I havent found anything similar to say, nuxtignore or something. I want it to ignore certain paths and not implement ssr there. 
I saw there was a <ClientOnly> </ClientOnly> option, but what if I have dozens or hundreds of components? Is there a way for it to ignore say a folder, but still have the paths added to the router?

**Comment by frandiox** [comment, marked as answer]
Hi! Right now, we don't have any feature to choose SSR vs SPA on a route basis. There are some alternatives:

- Use `<ClientOnly>` component around whatever you don't need in SSR.
- In your production server, before passing the route to `vite-ssr`, check the URL and just return `index.html` content if you don't want to do SSR. There's more info in [this issue](https://github.com/frandiox/vite-ssr/issues/70). If you want the same behavior during development, try providing your own dev server using `vite-ssr` in [middleware mode](https://github.com/frandiox/vite-ssr#middleware-mode).

Hope that helps!

**Reply by LagnerKol** [reply]
Thanks, will do it with ClientOnly for now then.
